editing/fx/passes/pact/harmonize.py

```diff
@@ -873,3 +859,69 @@ def __init__(self, ReplacementClass, ReplacementFunction = unwrap_mhsa_fun, modu
         tracer = LeafTracer(PACT_OPS)
         trace = partial(custom_symbolic_trace, tracer=tracer)
         super().__init__(op='call_module', target=tuple(pattern), replacement_fn = partial(unwrap_module_fun, wrapClass = ReplacementClass, modules=modules, unwrapFunction= ReplacementFunction), name=f"UNWRAP_PASS_{name}")
+
+def conv1d_replacement_fun(gm : fx.GraphModule, match : Match, *args, **kwargs):
+    modules = gm_modules(gm)
+    matched_modules = [modules[m.target] for k, m in match.nodes_map.items() if k.op == 'call_module'][::-1]
+    conv = matched_modules[0]
+    assert isinstance(conv, nn.Conv1d), f"conv1d_replacement_fun got bad match - expected nn.Conv1d, got {type(conv)}"
+    return PACTConv1d.from_conv1d(conv, **kwargs)
+
+def conv2d_replacement_fun(gm : fx.GraphModule, match : Match, *args, **kwargs):
+    modules = gm_modules(gm)
+    matched_modules = [modules[m.target] for k, m in match.nodes_map.items() if k.op == 'call_module'][::-1]
+    conv = matched_modules[0]
+    assert isinstance(conv, nn.Conv2d), f"conv2d_replacement_fun got bad match - expected nn.Conv2d, got {type(conv)}"
+    return PACTConv2d.from_conv2d(conv, **kwargs)
+
+class PactifyConvPass(SequentialPass):
```

Comment:
Let's give this a more indicative name - Something that brings out this is *homogeneous* quantization!
Maybe something like `HomogeneousConvFakeQuantizationPass`, but maybe shorter?

Reply:
Sure, see [this commit](https://github.com/pulp-platform/quantlib/pull/15/commits/d711f7d90be632c36942ebd4f41f5b0dd602f3af)